### ISSUE TITLE
Date archives picking up homepage meta description

### ISSUE THREAD

**Issue by kevinlisota** [sybrew/the-seo-framework#148]
Our date archives are showing the custom home page description in their meta description.

Not entirely sure why this is happening, but search engines don't like duplicate meta description. Google's search console even has a UI to check for this error.

**Comment by kevinlisota** [comment]
I am also seeing this on CPT archive pages. The home page meta description should only appear on the home page

**Comment by sybrew** [comment]
I'll look into it. Thanks! Will report back ASAP.

**Comment by sybrew** [comment]
Confirmed. Happening only when the home page is set as a blog. It only takes the auto-generated version.

Method `is_front_page_by_id()` should always return false when the home page is a blog.

**Comment by sybrew** [comment]
Hi @kevinlisota 

Thank you so much for looking into all these bugs, it seems that I made a mistake with the query implementation of 2.9.0.

It should all be fixed now as of b2c7b34.
[Download link](https://github.com/sybrew/the-seo-framework/archive/b2c7b3462380b6b3128f9b5324d988dc6e88a139.zip)

----

### Testing
If you wish to test the query output more easily, add this code to an active theme or plugin:
```
add_action( 'plugins_loaded', function() { if ( is_super_admin() ) {
	define( 'THE_SEO_FRAMEWORK_DEBUG', true );
}},0);
```

It will display all that's detected and used by The SEO Framework in the page footer (both admin and front-end), for the (network-)admins only.

![image](https://cloud.githubusercontent.com/assets/7662236/26760773/7055361a-4921-11e7-80f4-65b45eb7f068.png)

Arnold Schwarzenegger, Harrison Ford, Peter Weller and Fred Williamson will join you in that cause.
